### ISSUE TITLE
Make the design ti the ubuntu-touch OS

### ISSUE THREAD

**Issue by jv-barsuk** [bobo1993324/UDropCabin#14]
Icons are quite bad quality. The two column design could be replaced by three columns and the icons could be fitting the ubuntu-touch icons better. 

That would make the app more enjoyable.

**Comment by bobo1993324** [comment]
Yeah, the icons I found at the dropbox developer site is a bit off. I am thinking of using generic ubuntu minetype icons if that is what you mean by the icons.

**Comment by jv-barsuk** [comment]
Yes that's exactly what I mean.

**Comment by jv-barsuk** [comment]
The app looks very nice now. For me you can close this bug.
